source/localizable/cps_orders_api/_order_object.html.md.erb

```diff
@@ -0,0 +1,51 @@
+#### Order object
+
+Name                     | Type    | Description
+------------------------ | ------- | -----------------
+`code`                   | String  | Order code
+`date`                   | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order creation date
+`revenue`                | Double  | Total order price in euros
+`commission`             | Double  | Total order commission in euros
+`state`                  | String  | The current state of the order
+`rejection_reason`       | String  | The rejection reason of the order, if it was previously been declined
```

Comment:
We should mention that it's returned conditionally, as for `reject_options` field.

Reply:
Changed to `The declined reason of the order, if it has been previously rejected`